airflow/www/templates/airflow/tree.html

```diff
@@ -19,6 +19,12 @@
 
 {% extends "airflow/dag.html" %}
 {% block page_title %}{{ dag.dag_id }} - Tree - {{ appbuilder.app_name }}{% endblock %}
+{% from 'appbuilder/loading_dots.html' import loading_dots %}
+
+{% block head_meta %}
+  {{ super() }}
+  <meta name="dag_id" content="{{ dag.dag_id }}">
```

Comment:
Should this maybe go in the base dag.html template, and then get pulled out of graph.html?